File: 1-Bimestre/exercícios/2026-03-13 - somar dois números/emTypeScript/somar.js
"use strict";
window.funcaoSomarDoisNumeros = () => {
    // Entrada de dados para a função
    const valorA = parseFloat(document.getElementById("inputValorA").value);
    const valorB = parseFloat(document.getElementById("inputValorB").value);
    // Processamento
    const soma = valorA + valorB;
    document.getElementById("resultado").value = soma.toFixed(2).toString();
};
// Opcional: Adicionar evento para evitar problemas de carregamento
window.addEventListener('DOMContentLoaded', () => {
    const botao = document.getElementById("botaoSomar");
    if (botao) {
        botao.onclick = window.funcaoSomarDoisNumeros;
    }
});

File: 1-Bimestre/exercícios/2026-03-13 - somar dois números/emPython/somar.py
def funcaoCalcular():
    # Entrada de dados (equivalente ao parseFloat do JavaScript)
    valorA = float(input("Digite um valor para A: "))
    valorB = float(input("Digite um valor para B: "))
    
    # Processamento
    soma = valorA + valorB
    
    # Saída (equivalente ao innerHTML)
    print(f"resposta: {soma}")

# Executar a função
funcaoCalcular()